### DISCUSSION TITLE
lastmod doesn't working

### DISCUSSION THREAD

**Discussion by monomable** [cotes2020/jekyll-theme-chirpy#1594] [Q&A] [answered]
### Checklist

- [x] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

all of my posts doesn't showing lastmod in my github pages
lastmod is working in my local pages but github pages doesn't working

### Operations you have already tried

update jekyll, update chirpy, reload file /commons.min.js

**Comment by steveng57** [comment]
I had the same problem, and could not get it to work on GH pages.  I believe that it doesn't work there because GH Pages will not run arbitrary plug-ins, only the ones in their pre-approved list.  This is for security reason, which makes sense.

What I ended up doing was just setting the var `last_modified_at` in the Front Matter of a post... like so...

`last_modified_at: 2024-02-17 18:32:00 -0500`

Hope that helps.

**Reply by monomable** [reply]
i'm fixed it

**Comment by monomable** [comment, marked as answer]
I'm fixed it

i'm trying remove file
`/.github/workflows/jekyll.yml`

and 
i'm add file
`/.github/workflows/pages-deploy.yml`
in jekyll-starter

https://github.com/cotes2020/chirpy-starter/blob/main/.github/workflows/pages-deploy.yml

and then it normally running
